### DISCUSSION TITLE
Looking for advice on install troubleshooting

### DISCUSSION THREAD

**Discussion by equ8r** [Budibase/budibase#5882] [Help]
I've hit a wall on launching Budibase in Docker... I just lack some context and haven't been able to find it. I use docker on Ubuntu, but for this purpose I am running it in Windows. 

I set up docker on the windows box without any issues, and spun up containers with postgres and pgadmin. All is well so far... just like running them in ubuntu.

I downloaded and ran the YAML file, and containers were created for the following:

![image](https://user-images.githubusercontent.com/105300471/168378903-bfb3b9dd-6f64-4fa5-b4be-b989ac36f3b6.png)

The containers that show a status of 'restarting' try to restart several times and present various error messages that all seem to be authentication related issues for the relevant services.  

For instance - bbapps is referencing "TypeError: JwtStrategy requires a secret or key"....

I'm a bit at a loss for next steps here. Anyone able to kick me in the right direction?

**Comment by equ8r** [comment]
Here's a short screen recording to help visualize what I'm seeing...

https://user-images.githubusercontent.com/105300471/168400890-a824e508-2379-4d29-b478-3117eda6d549.mp4

**Reply by shogunpurple** [reply]
@equ8r it seems like you are missing your .env file.

https://raw.githubusercontent.com/Budibase/budibase/master/hosting/.env

You need to have this in the same directory as your docker-compose.yaml.

It’s mentioned in the setup docs here:

https://docs.budibase.com/docs/docker-compose

**Comment by equ8r** [comment]
Much appreciated on the kick @shogunpurple. I had the file in that directory, but it looks like windows slugged it as 'env.txt' when I downloaded it. I changed the name back to ".env" and everything fires up.
